lib/config.js

```diff
@@ -36,20 +36,18 @@ config.mongodb.authentication = {
   // used by MongoDB < 3.0
   // authMechanism: 'MONGODB-CR'
 };
-
 // this is used when making connections to the database
 config.mongodb.connectOptions = {
   useUnifiedTopology: true,
   serverSelectionTimeoutMS: 30000,
   autoReconnect: false,
   useNewUrlParser: true,
-  // the db to authenticate against
-  authSource: undefined,
-  // we use the default for ssl
-  // set this to true if you want to require ssl
-  ssl: undefined,
   // promotes binary BSON values to native Node.js buffers
   promoteBuffers: true,
+  // FIXME: we need development and production servers to use SSL
```

Comment:
Why is this a FIXME?

Reply:
ok so in order to make SSL: true a default now it would be a breaking change (especially in development where the local mongodb server people use are usually not using SSL). I've talked to @mattcollier about this and basically we need to revise the docker images used for mongodb in development to take in a PEM ed25519 certificate for SSL.

Reply:
Just to make this more clear if `ssl: true` is an option, then mongodb servers with out ssl configured will hang and fail to connect. So turning it to true by default is definitely breaking.

Reply:
That's fine, but this doesn't sound like a FIXME. It should just be a comment about other these options that can be set. The comment should remove the "FIXME" portion and then indicate that the options should be set in production and that they can't be set to undefined for `<enter reason here>`. If we need another comment that indicates a future breaking change may set them to some other default, that can go there too.

Reply:
I changed it to recommend setting either `tls` or `ssl` to true in production: https://github.com/digitalbazaar/bedrock-mongodb/pull/89/commits/60ee8241d1355bfcc535ec93abaab4154422a4fe